requirements.txt

```diff
@@ -11,4 +11,5 @@ seaborn>=0.11.1
 category_encoders>=2.0.0
 statsmodels >= 0.12.2
 imbalanced-learn>=0.8.0
+pmdarima==1.8.0
```

Comment:
Can we do `pmdarima>=1.8.0`?

Reply:
I can give it a shot, I was worried about some of their latter releases because they started pinning dependencies and I was worried how it would impact us

Reply:
Yeah I think there's a pinning on NumPy that causes an error in the tests unfortunately

Reply:
Got it, that makes sense to me then. Thanks!